### PR TITLE
add list of public seed nodes & disclaimer

### LINKED ISSUE #115: Create a list of community seed nodes
As @viraptor mentions (https://radicle.community/t/public-seed/1060) it would be valuable to have a list of community seed nodes that others can connect to... Maybe a section on docs.radicle.xyz?

### PR DESCRIPTION
**Pull request by abbey-titcomb**
Added language on seed nodes and linked the radicle.community thread. Closes #115 

Signed-off-by: abbey titcomb <abbeytitcomb@gmail.com>

### CODE REVIEW

**Review by xla — CHANGES_REQUESTED** [review]
Can we make sure formatting is correct (line lengths). And should we link to the radicle.community thread for seed announcments.

**Comment by abbey-titcomb** [comment]
> Can we make sure formatting is correct (line lengths). And should we link to the radicle.community thread for seed announcments.

I did link to it see blockquote in "adding a seed node"

What do you mean by formatting?

File: docs/using-radicle/adding-a-seed-node.md [hunk 1]
```diff
@@ -8,8 +8,11 @@ to connected peers. When you add a seed to Upstream, it will
 automatically track you and share your data across its network of other
 connected peers [^1]. This increases the availability of your data throughout
 the network, while making it easier to find others data as well. Upstream is
-preconfigured with a Radicle operated seed to bootstrap your connectivity. If
-you have removed the default seed and need to re-add it or would like to add a
+preconfigured with a Radicle operated seed to bootstrap your connectivity. Adding more seed nodes to your Upstream configuration is a great way to improve data availability and redundancy. 
+
+> To discover seed nodes, please refer to this <a href="https://radicle.community/t/list-of-public-seeds/1155">list of public seed nodes</a>.
```
Comment (FintanH) [inline docs/using-radicle/adding-a-seed-node.md:13]: Why are you using `<a></a>` rather than markdown linking?

**Review by FintanH — COMMENTED** [review]
Looks good. 

Formatting means sticking to set column width. Are we using 80, 100, or something else?

**Comment by FintanH** [comment]
> Formatting means sticking to set column width. Are we using 80, 100, or something else?

This means that git diffs will look better in the long run as we change particular lines rather than a big long line :)

File: docs/using-radicle/adding-a-seed-node.md (hunk 1 above)
Comment (abbey-titcomb) [reply docs/using-radicle/adding-a-seed-node.md]: oops forgot, fixed

**Comment by abbey-titcomb** [comment]
> Looks good.
> 
> Formatting means sticking to set column width. Are we using 80, 100, or something else?

"If you or Christoph have 80 character lines, you'll get possibly ugly
wrapped output. Tough. That's _your_ choice. Your hardware limitations
shouldn't be a pain for the rest of us.

Longer lines are fundamentally useful. My monitor is not only a lot
wider than it is tall, my fonts are universally narrower than they are
tall. Long lines are natural."

jk. fixed. 

tack

**Review by xla — APPROVED** [review]
🏄 👗 ⛹ 💬

**Pull request merged**